infra/terraform/test-org/org/locals.tf

```diff
@@ -752,5 +752,17 @@ locals {
       owners      = ["anaik91", "imrannayer"]
       topics      = local.common_topics.db
     },
+    {
+      name            = "terraform-pubsub-integration-golang"
+      org             = "GoogleCloudPlatform"
+      short_name      = "pubsub-golang-app"
```

Comment:
both of these need owners and groups.  Example:
      owners      = ["shabirmean", "Mukamik"]
      groups      = ["dee-platform-ops", local.jss_common_group]

Reply:
thank you @donmccasland !